app/models/user.rb

```diff
@@ -275,6 +277,26 @@ def current_immersive_context_and_content_at(path_item)
     end
   end
 
+  def formal_first_name
+    verified_first_name || first_name
+  end
+
+  def formal_last_name
+    verified_last_name || last_name
+  end
+
+  def formal_full_name
+    "#{formal_first_name} #{formal_last_name}"
+  end
+
+  def certificates_in(organization)
```

Comment:
It isn't expensive but nevertheless why aren't we using an AR query for this one too?

Reply:
Also it would be great to have Organization.current as default for organization.
Maybe then the method should be called `certificates_in_organization`